src/main/java/seedu/address/model/module/student/StudentIdContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,27 @@
+package seedu.address.model.module.student;
+
+import java.util.function.Predicate;
+
+/**
+ * Tests that a {@code Student}'s {@code Name} matches any of the keywords given.
+ */
+public class StudentIdContainsKeywordsPredicate implements Predicate<Student> {
+    private final String keyword;
+
+    public StudentIdContainsKeywordsPredicate(String keyword) {
+        this.keyword = keyword;
+    }
+
+    @Override
+    public boolean test(Student student) {
+        return keyword.equals(student.getStudentId().value);
```

Comment:
if it is checking if two strings are equal would it be better to name this class smt like StudentIdEqualKeywords... instead?

Reply:
Yes I wanted to change to that also, but seems like they use the class Predicate for these things

Reply:
I mean like StudentIdEqualKeywordsPredicate?

Reply:
Ah yes cannn I’ll change it!! Thanksss hehe